README.md

```diff
@@ -63,17 +63,20 @@ This section outlines the environment variables required for the application. Be
 | `DB_URI`          | Postgres connection string                | `postgresql://user:password@localhost:5432/dbname`              |
 | `MAINNET_RPC_URL` | Provider URL for the Ethereum mainnet RPC | `https://mainnet.infura.io/v3/8238211010344719ad14a89db874158c` |
 
-- **SPACES format**: Spaces should be defined by the ENS and the starting block, with the following format: `kleros.eth:3000000`. Multiple spaces can be present, separated by commas.
+- **SPACES format**: Comma separated list of spaces to follow. The spaces are defined by the ENS, starting block and (optionally) the Reality module contract address. These three values should be colon separated, following the format `ens:block<:reality module address>`. When the reality module address is not provided, the bot tries to auto-detect it. Examples:
```

Comment:
@wadader I will make it snake case.

I introduced the `<>` to indicate which part is optional. Sometimes the `[]` is used with the same meaning, not sure if that one would be clearer. Maybe I should just place `ens:block or ens:block:reality_module_address formats` instead of a syntax with a conditional instead.

Reply:
I was unfamiliar with the `<>` convention to denote `optional`. 

If it is a common convention, please proceed as normal and I will add it in some corner of my brain.

Reply:
@wadader I think the underlying feedback was good: it can be made way easier to read and less prone to confusions, so I'll change the wording as said.

 `[]` is more common than `<>` now that I think about it.

Reply:
@wadader changed, do you find it better now?

Reply:
Significantly easier, thanks